### DISCUSSION TITLE
P7 to P8 conversion error

### DISCUSSION THREAD

**Discussion by jfnc1998** [R3S3t9999/DoVi_Scripts#40] [General]
The script completes and behaves like it should, but the output file can't be played after about 3-4 minutes. It's happened on two files, consistently, the same error, tried the beta and redownloading the tools pack/script.
Both files are MEL, but one is CMV4.0 and the other CMV2.9.
Didn't notice this happening on older versions of the tools/scripts.
Can provide any additional info.

**Comment by R3S3t9999** [comment]
Hi, it sounds like a player issue. 
I converted and watched hundreds of P7 to P8 rip with my Ugoos/X800m2 or C2 internal player.

**Comment by jfnc1998** [comment]
I'm playing it on a Ugoos, also tried VLC on windows and infuse ATV, just to check if it was a DV issue or if it was in the base layer too and it happens on all of them exactly the same regardless of DV or HDR10.
I've converted hundreds of files and the tool has worked great, but once I updated to 3.0.2 and got these two files that I had this error.
If I leave them as MEL they play no problem on the Ugoos with DV and HDR10 on VLC, as expected.
Is there any thing I could look for in the files? Or some test I could perform on them?

**Comment by R3S3t9999** [comment]
Nothing changed in how P7 is converted to P8 in 3.0.2 so I don't know. 
Which movies? What's your source,  untouched bluray folder?  Does it do this for all the movies or only two specific files?

**Comment by jfnc1998** [comment]
It's only these two specific files so far, both are mkv remuxes, one is The Running Man (1987) from framestor with CMV2.9 MEL (Sourced from BHD UHD-BR), and the other is Fist of Fury (1972) from cinephiles with CMV4.0 MEL (Sourced from Arrow release). No problems before converting on either of the players, even remuxing to add/remove subs/audio keeps the file playable, its only until it becomes a P8. 
I can provide whatever info or samples just to rule out something on the script, but right now I'm more inclined to believe theres something up with these two files, since it's never happened before, its just weird that I got these two errors on the same week just after updating DV scripts and tools folder.

**Comment by R3S3t9999** [comment]
thanks, I'll check those two films but you could try to disable the ffmpeg piping (line 99)

::choose if you dont want to use ffmpeg piping for demuxing. You might have to disable if you notice issues.  In 4-1:  only valid for the mkv version and when the script is set to mux all audio/subs (default = YES)
set ffmpeg_pipe=YES

**Comment by jfnc1998** [comment]
I'll give that a try and update if it fixes the issue.
Thank you.

**Comment by R3S3t9999** [comment]
I can replicate the issue with Running Man and disabling ffmpeg piping fixed the issue for me.
ffmpeg piping speeds up the process a lot but on very rare occasions, it can cause this kind of problem.

**Comment by jfnc1998** [comment]
Yup, just got done with Fist of Fury with piping disabled, flawless now. I was just reading the old releases and I guess this was introduced on 2.0.4 with the speed increase to 4-1, I was probably running an earlier version before I made the jump to 3.0.2. 
Anyway, I'll just keep the piping off, thank you so much for the fast support.
